packages/cli/src/cli.ts

```diff
@@ -0,0 +1,36 @@
+import { EntryPoint__factory, SimpleWalletDeployer__factory } from '@account-abstraction/contracts'
+import { SimpleWalletAPI } from '@account-abstraction/sdk'
+import { DeterministicDeployer } from '@account-abstraction/sdk/dist/src/DeterministicDeployer'
+import { Wallet } from 'ethers'
+import hre from 'hardhat'
+
+const epAddress = '0x2167fA17BA3c80Adee05D98F0B55b666Be6829d6'
+
+async function main (): Promise<void> {
```

Comment:
What is this CLI, exactly? Is it a deployer, or a sender, or what? I think all CLI commands must be a named Commander commands.

Reply:
commited by mistake. already removed.